classes/PodsUI.php

```diff
@@ -3076,14 +3137,30 @@ public function manage( $reorder = false ) {
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 									<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'text', $options );
+									// Prevent p div issues.
+									echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: 9ca67974c5c79f609547b6987d791194</sub>